### ISSUE TITLE
Clicking PR title looses focus

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode-pull-request-github#5332]
![Recording 2023-10-11 at 05 28 22](https://github.com/microsoft/vscode-pull-request-github/assets/900690/fc5f1c87-e4eb-416d-95c7-14fa2941563c)

**Comment by alexr00** [comment]
This is so that that screen readers can know that something has opened. @meganrogge FYI.

**Comment by bpasero** [comment]
@alexr00 all trees/lists should behave consistently:
- single mouse click: focus remains
- double click: focus moves
- enter/cmd+down: focus moves

**Comment by alexr00** [comment]
Hmm. Looks like the "Open Description" command is special. To pass through the click modifiers, we must use the `vscode.open` or `vscode.diff` commands. We don't pass through click modifiers for other commands, and we don't have a way for commands to opt into this. "Open Description" cannot use `vscode.open`, because it opens a webview.

Because of this, I actually can't handle https://github.com/microsoft/vscode-pull-request-github/issues/5333 right now either.

It looks like we need a way to let commands opt in to receiving click modifiers.

**Comment by alexr00** [comment]
@mjbvz is there a way I can open a webview by using `vscode.open`?

**Comment by mjbvz** [comment]
@alexr00 I don't think so. We don't expose a uri for the webview to extension but every `WebviewInput` does have a unique uri. Maybe if you can get a hold of that `vscode.open` would work but I've never tested this

**Comment by alexr00** [comment]
The uri changes every time the webview is opened, so it doesn't seem like this will work. Closing as out of scope.

**Comment by hediet** [comment]
I don't understand why this is out of scope.
Is this because webviews always get focused when they open?

**Comment by alexr00** [comment]
I marked it out of scope because unless something changes with how webviews are allowed to be opened, I don't intend to fix it. We need the current behavior for accessibility.

**Comment by hediet** [comment]
> We need the current behavior for accessibility.

Can you only do it when the screen-reader mode is turned on?

I'm quite sure this wouldn't break anyones workflow, but provide a more consistent experience.

**Comment by alexr00** [comment]
Yes, only doing this when screen-reader on seems reasonable.

**Comment by alexr00** [comment]
@meganrogge is there any way I can tell from an extension whether the user has accessibility mode turned on? I see the setting has an `auto` option, which means that I can't know for sure just based on the setting.

**Comment by meganrogge** [comment]
We don't have API for that currently.

**Comment by meganrogge** [comment]
Could you check the context key from the extension though?

**Comment by alexr00** [comment]
I would need to be able to check the context value dynamically, not in the package.json, as this command is set on the tree item and not through menus in the package.json.

We don't have an API command to get a context value, just the one to set a context value.

**Comment by alexr00** [comment]
Moving to the backlog as this requires API.
